examples/novatel/command_encoding/command_encoding.cpp

```diff
@@ -25,10 +25,12 @@
 // ===============================================================================
 
 #include <chrono>
+#include <cstdio>
+#include <cstdlib>
 #include <filesystem>
 
-#include "src/decoders/novatel/api/commander.hpp"
-#include "src/hw_interface/stream_interface/api/outputfilestream.hpp"
+#include <decoders/novatel/api/commander.hpp>
+#include <hw_interface/stream_interface/api/outputfilestream.hpp>
```

Comment:
why angled brackets?

Reply:
Yeah, I like to think of example code files as standalone code projects that should work independently from the main code base, if possible. Double quotes would be ok as well, though.